theses/README.md

```diff
@@ -4,3 +4,8 @@
 ## Incomplete symbolic execution with Monster: Solver by Alexander Lackner, University of Salzburg, Austria, 2021 ([PDF](https://github.com/cksystemsgroup/monster/releases/download/v0.1.0/Incomplete_Symbolic_Execution_with_Monster_Solver.pdf), [Release](https://github.com/cksystemsgroup/monster/releases/tag/v0.1.0))
 
 Monster is a symbolic execution engine that can use different SMT solvers to detect bugs in C* programs. Solvers are the bottleneck of symbolic execution engines, therfore a faster solver directly leads to faster symbolic execution. Since symbolic execution is incomplete by design, we took this a step further, implementing and designing an incomplete solver based on Aina Niemetz's work. Our solver being incomplete does mean that it can not decide if an SMT formula is unsatisfiable. We use this to our advantage by not searching the entire set of assignments but instead a smaller subset. This should lead to a faster and less accurate solver. We reached the following results benchmarking against Boolector and Z3, while Monster was able to be competitive when shorter and simpler programs were being fed into the engine, it struggled with long and more complex programs with no or hard-to-find solutions meaning its performance dropped significantly. It did however always return the same results as its counterpart, showing that its accuracy is just as high which is a feat itself given the short timeframe and being constructed by three bachelor students.
+
+
+## Incomplete symbolic execution with Monster: Solver by Alexander Lackner, University of Salzburg, Austria, 2021 ([PDF](https://github.com/cksystemsgroup/monster/releases/download/v0.1.0/Incomplete_Symbolic_Execution_with_Monster_System.pdf), [Release](https://github.com/cksystemsgroup/monster/releases/tag/v0.1.0))
```

Comment:
This doesn't look right...

Reply:
Yes that was a pretty stupid oversight. Corrected now.